Kempe(Linear)/Kempe(Linear).py

```diff
@@ -0,0 +1,90 @@
+# This code is contibuted by Riyazul555
+
+# Time complexity = O(N * (N + M))  N represents number of vertices
+# Space complexity = O(N + M)       M represnts number of  edges
+
+
+def graph_coloring():
+    n_max = 25
```

Comment:
I think the same suggestions from #51 apply. There is no need for declaring a maximum value for N, since lists are dynamically allocated (compared to C-style arrays).